docs/Authenticating-Users-via-LDAP.md

```diff
@@ -13,8 +13,6 @@ In this setup LDAP will be used as a simple Identity Provider (IDP) for the auth
 
 * **authentication**:  a means of verifying that a user is who they purport to be.  Authentication is performed by the identify provider, by extracting the user name and password provided in a login request, and matching this with information stored in its repository.
 
-* **authorization**:  defines resources a user can access.  When authorization is turned *on* with cBioPortal, users can only access cancer studies they are specifically authorized to view.  This enables one to store multiple cancer studies within a single instance of cBioPortal, but provide fine-grained control over which users can access which studies.  Authorization is implemented within the core cBioPortal code, and *not* the identify provider.
```

Comment:
I think maybe the comments about permissions per user to the authentication section